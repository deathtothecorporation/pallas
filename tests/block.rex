a=b
b=c
; hi
g=g

| x
g=g
g=g

g=g
