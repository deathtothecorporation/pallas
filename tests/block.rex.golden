a=b

b=c

g=g

| x g=g
g=g

g=g
